lib/lbt/resources/LocatorResourcePool.js

```diff
@@ -3,8 +3,8 @@ const LocatorResource = require("./LocatorResource");
 
 
 class LocatorResourcePool extends ResourcePool {
-	constructor() {
-		super();
+	constructor(parameters) {
```

Comment:
AFAIK you can also just completely remove the constructor.

Reply:
kk (y) that's even better, I give it a try